tests/user/test_user_debug_logging.py

```diff
@@ -184,6 +184,54 @@ def test_add_user_request_debug_log_passphrases_and_passwords_get_redacted_on_er
         self.assertNotIn(self.test_passphrase, error_log)
         self.assertNotIn(self.test_password, error_log)
 
+    def test_add_user_request_debug_log_added_secrets_get_redacted_on_success(
+        self,
+        call_racf_mock: Mock,
+    ):
+        self.user_admin_local = UserAdmin(
```

Comment:
Since you are creating a local `UserAdmin` object, you do not need to define it as a class attribute. You can just do this:

```python
        user_admin = UserAdmin(
            debug=True, additional_secret_traits=["omvs:uid"]
        )
```

Reply:
Good point, change made.